Dockerfile

```diff
@@ -6,15 +6,23 @@ RUN set -ex && \
     apt-get install -y --no-install-recommends \
     python3-pip
 
+# Install Python dev dependencies.
```

Comment:
Testing locally, I had to move lines 14-18, the apt-get for curl and build-essential, above 10-12, or else it failed with `E: Failed to fetch http://deb.debian.org/debian/pool/main/p/perl/perl-modules-5.32_5.32.1-4%2bdeb11u2_all.deb  404  Not Found [IP: 151.101.22.132 80]` (not sure why). Can we swap the order?